ios/Classes/core/PMManager.h

```diff
@@ -16,8 +16,13 @@ typedef void (^ChangeIds)(NSArray<NSString *> *);
 #import "PMResultHandler.h"
 #import "PMConvertProtocol.h"
 
+#define PM_VIDEO_CACHE_PATH @".video"
+#define PM_IMAGE_CACHE_PATH @".image"
+#define PM_FULL_IMAGE_CACHE_PATH @"flutter-images"
```

Comment:
Should we use `pm-images`?

Reply:
There is a historical reason here, previously `flutter-images` was used, if it is changed to `pm-images` here, the old images will not be deleted in method.